software/introduction.qmd

```diff
@@ -145,19 +145,20 @@ that fit these principles:
 ### Users
 
 The Seedcase framework is developed and designed with four users and
-uses cases in mind. The four users, which are described in more detail
-in the [Context section](context.qmd), are:
+their associated use cases in mind. The four users, which are described
```

Comment:
I guess, technically, these 4 are roles for users rather than the users themselves, but maybe we don't have to be that pedantic in the introduction, so I didn't change it here.

Reply:
I think it makes sense to change it to "user roles" here.

Reply:
I've accepted Luke's suggestion below which was also about this :)